### PR TITLE
docs(examples): add kaggle example

### PR DESCRIPTION
**Pull request by jitingxu1**
Add example: Home Credit - Credit Risk Model Stability [link](https://www.kaggle.com/competitions/home-credit-credit-risk-model-stability/overview)

### CODE REVIEW

**Comment by deepyaman** [comment]
To be updated based on blog post content/review.

File: examples/Home Credit - Credit Risk Model Stability.ipynb
Comment (deepyaman) [inline examples/Home Credit - Credit Risk Model Stability.ipynb:1]: Nit: I think `Using IbisML and DuckDB for a Kaggle competition.ipynb` is a more descriptive title, like you have in your post. :)

**Review by deepyaman — APPROVED** [review]
LGTM!

Couple minor things I noticed (other than the suggestion to rename):

* You have one more instance of `ROOT = Path("/Users/claypot/Downloads/home-credit-credit-risk-model-stability")`; do you just want to rename `claypot` to `jitingxu` or something there?
* No need to do this in this PR, or with any priority, but I think it would be _ideal_ to use `%%bash` cells and render the output instead of \`\`\`bash\`\`\` in Markdown (and writing out the outputs).

Feel free to merge as is, or with the suggested notebook rename + claypot->jitingxu or something; leave it up to you!

File: examples/Home Credit - Credit Risk Model Stability.ipynb
Comment (jitingxu1) [reply examples/Home Credit - Credit Risk Model Stability.ipynb:1]: Yeah, I will change the title and minor thing you mentioned in the above thread.

**Pull request merged**